scripts/build-image

```diff
@@ -42,19 +42,21 @@ def build(image, image_name, image_tag, config, pull=False):
 
     rc = 0
     pull_base = ''
+    args = config.get(image, 'build_args') if config.has_option(image, 'build_args') else ''
+
     if pull:
         pull_base = '--pull'
     if is_multistage(image):
         # if this is a multistage build and it follows the conventions, tag the builder image
         # otherwise, a prune will remove the layers used during the builder phase and subsequent
         # builds will take longer than required
         rc = image_operation(
-            'docker build {pull_base} --compress -t {image_name}-builder:{image_tag} -f docker/{image}/Dockerfile --target builder .'
-                .format(pull_base=pull_base, image_name=image_name, image=image, image_tag=image_tag))
+            'docker build {pull_base} --compress -t {image_name}-builder:{image_tag} -f docker/{image}/Dockerfile --target builder {args}.'
```

Comment:
should there be a space between `{args}` and `.`?

Reply:
Yes